src/configuration/app-containers.md

```diff
@@ -142,31 +143,31 @@ build:
 
 ### Access
 
-The `access` define the user roles who can log in via SSH to the
-environments they have access to.
+The `access` defines the user roles who can log in via SSH to the
+environments they have permission to access.
 
-Possible values are:
+Possible values are below:
 
 * ssh: admin
 * ssh: contributor
 * ssh: viewer
 
 ### Relationships
 
-The `relationships` defines how services are mapped within your
+The `relationships` define how services are mapped within your
 application.
 
 The left-hand side is the name of the relationship as it will be exposed
 to the application in the *PLATFORM_RELATIONSHIPS* environment
 variable. The right-hand side is in the form
-`<service name>:<endpoint name>`. where "service name" comes from the
+`<service name>:<endpoint name>` where "service name" comes from the
  `.platform/services.yaml` and  "endpoint name" should be the same as the
- value of "type"  declared in that same file.
+ value of "type" declared in that same file.
 
 > **Note**
-> In the first  example above you could very well have something
+> In the first example above, you could very well have something
```

Comment:
As above, verbal pause and written comma do not always correlate, because English.